### DISCUSSION TITLE
Agenda for the Flatcar Community Call on Tuesday, 10th of May 2022 17:30 CEST / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#726] [Flatcar Office Hours]
## Agenda for the Flatcar Community Call on Tuesday, 10th of May 2022  17:30 CEST / 11:30am EDT / 8:30am PDT

* Zoom link:  https://us06web.zoom.us/j/84336116610
  - Meeting ID: 843 3611 6610
  - Passcode: 444888
- Youtube live stream / recording: [link](https://www.youtube.com/watch?v=JS3PeGrDjkg&ab_channel=Kinvolk)
- Community Calls calendar (all future calls): [link](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [link](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)

## Welcome
- Brief intro/check-in of the new participants in the Zoom call. Please introduce yourself and share what brings you here today.

## News
- LTS moving to community
- New features in latest community release 
- Meet the Flatcar team at Cloud Native Rejekts and KubeCon in Valencia

## Release planning
- Ongoing and upcoming releases (https://github.com/orgs/flatcar-linux/projects/7/views/8)

## Q&A
- Questions from community participants, answered by the Flatcar maintainers

**Comment by t-lo** [comment]
News: We requested a channel in the [Kubernetes Slack](https://github.com/kubernetes/community/pull/6639) and things are looking good! We'll maintain this channel alongside Matrix as we feel both cater separate audiences. The maintainers will be reachable for cloud-native folks on Slack, and chat with Linux OS folks on Matrix.

**Comment by pothos** [comment]
I think I'll skip attending today if someone can cover the LTS introduction.
The LTS channel is now publicly available, and is almost like the Stable channel with larger timespans, but has an additional concept of "streams" which refer to the major releases and the bugfix updates for the major releases. A new major release happens around once per year. Here are instructions on how to control the LTS upgrade from one major version to the next:
https://www.flatcar.org/docs/latest/setup/releases/switching-channels/#freezing-an-lts-stream
https://www.flatcar.org/docs/latest/setup/releases/switching-channels/#use-a-personal-update-server

**Reply by t-lo** [reply]
Happy to cover for you if you prefer to pass, the call is really late for your timezone.

**Reply by pothos** [reply]
Thanks :)

**Comment by miao0miao** [comment]
keeping an eye on issues posted from community members:

## Issues Opened the last 30 days:
* [#733][Olivier Bourdon](https://github.com/obourdon):  Moving from stable 3033.2.4 to 3139.2.0 made kitchen tests fail 
* [#731][Christian Bianchi](https://github.com/whites11) :  networkd units differ between ignition.json and filesystem
* [#729][Dmitri Fedotov](https://github.com/defo89) :  Beta 3185.1.0: ignition fails to create partition on second disk (vmware)
* [#722][Rodny Molina](https://github.com/rodnymolina):  cgroupv1-revert solution breaks procfs mounts within user namespaces
* [#721]Yusuke Suzuki:  Cilium with loadBalancer.acceleration native fails on 3139.2.0
* [#716]Tony Gosselin:  [RFE] Arm64 Support for Azure Kubernetes Service (AKS)

## Issues Closed the last 30 days:
* [#722]Rodny Molina:  cgroupv1-revert solution breaks procfs mounts within user namespaces
* [#714][Foivos Filippopoulos](https://github.com/ffilippopoulos) :  oem-gce.service is broken after upgrading to flatcar 3139.2.0
* [#710][Joe Gooch](https://github.com/goochjj):  newest beta 3185.1.0 does not boot
* [#707][Jan-Otto Kröpke](https://github.com/jkroepke):  Flatcar on EC2 sets a incorrect os hostname
* [#706][Chris Swan](https://github.com/cpswan):  How long to wait between update_engine_client -reset_status and -check_for_update ?
* [#220] [/v\atthew L Daniel](https://github.com/mdaniel):  Add AWS IMDSv2 mode support for ignition and afterburn

**Comment by miao0miao** [comment]
reach out if you are Valencia and want to join the party, the number of tickets is limited 

https://www.eventbrite.com/e/giant-party-1-tickets-298038760957
